include/kyosu/types/impl/arithmetic.hpp

```diff
@@ -95,4 +96,35 @@ namespace kyosu::_
   {
     return C{kumi::map([](auto const& e) { return eve::frac(e); }, c)};
   }
+
+  template<concepts::cayley_dickson C>
+  KYOSU_FORCEINLINE constexpr
+  auto dispatch(eve::tag_of<kyosu::minus> const&, C const& c) noexcept
+  {
```

Comment:
if this is just that, you can actually just put it in the deferred_call in the minus object

Reply:
give me some details:
do I simply put typename in place of ordered_value ?

Reply:
done